### PR TITLE
Version instrumentation

### PR DESCRIPTION
**Pull request by thomas-gerber**
Version instrumentation

Relies on the fact that `docker image ls` is sorted by creation date (i.e. the same ordering for what image is used for a given repository/tag pair).

Also adds a `source` instrumentation that we can use for campaigns.

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -27,6 +27,9 @@ function parseFlags() {
             --email)
                 email_override=$2
                 shift 2 ;;
+            --source)
+                source=$2
```
Comment (tovbinm) [inline start.sh:31]: let's make sure the `source` argument is optional

Comment (thomas-gerber) [reply start.sh:31]: @tovbinm it is optional (same as `email`).

Comment (thomas-gerber) [reply start.sh:31]: added default (although it worked without)

File: start.sh [hunk 2]
```diff
@@ -53,9 +56,18 @@ main() {
 
   export FAROS_EMAIL=$EMAIL
 
+  if [[ -n "$source" ]]; then
```
Comment (tovbinm) [inline start.sh:59]:
```bash
if [[ -n "$source" ]]; then
    SOURCE=$source
else
    SOURCE="unknown"
fi
```

**Review by tovbinm — APPROVED** [review]

**Pull request merged**